kit/foundation/docs/main.tf

```diff
@@ -0,0 +1,160 @@
+        "<!-- error locating compliance control ${s.control} -->"
+      )])
+    )
+  }
+}
+
+resource "null_resource" "copy_template" {
+  triggers = {
+    output_dir         = var.output_dir
+    template_dir_files = join(" ", fileset(var.template_dir, "**/*")) # since we use symbolic links, we don't care for file content
+  }
+
+  provisioner "local-exec" {
+    when = create
+    # copy files as symbolic links, this means we can change them in the source dir and live reload will work!
+    command = "mkdir -p ${var.output_dir} && cp --archive --recursive --symbolic-link ${var.template_dir}/* ${var.output_dir}"
+  }
+
+  provisioner "local-exec" {
+    when = destroy
+    # remove symbolic links
+    command = "cd ${self.triggers.output_dir} && rm -f ${self.triggers.template_dir_files}"
+  }
+}
+
+resource "null_resource" "copy_compliance" {
+  triggers = {
+    output_dir         = var.output_dir
+    template_dir_files = join(" ", fileset(local.compliance_dir, "**/*")) # since we use symbolic links, we don't care for file content
+  }
+
+  provisioner "local-exec" {
+    when = create
+    # copy files as symbolic links, this means we can change them in the source dir and live reload will work!
+    command = "mkdir -p ${var.output_dir}/docs/compliance && cp --archive --recursive --symbolic-link ${local.compliance_dir}/* ${var.output_dir}/docs/compliance"
+  }
+
+  provisioner "local-exec" {
+    when = destroy
+    # remove symbolic links
+    command = "cd ${self.triggers.output_dir}/docs/compliance && rm -f ${self.triggers.template_dir_files}"
+  }
+}
+
+resource "local_file" "module_docs" {
+  depends_on = [null_resource.copy_template]
+  for_each   = local.terragrunt_modules
+
+  filename = "${var.output_dir}/docs/${each.key}.md"
+  content = join(
+    "\n\n",
+    compact([
+      # documentation_md
+      try(data.terraform_remote_state.docs_azure[each.key].outputs.documentation_md, "*no `docmentation_md` output provided*"),
+      # by convention, we expect that a platform module uses the same kit module name so we use that to lookup compliance statements
+      "## Compliance Statements",
+      coalesce(
+        try(local.kit_module_compliance_md[local.terragrunt_to_kit_modules[each.key]], null),
+        "*no compliance statements provided*"
+      )
+    ])
+  )
+}
+
+# todo: not sure we want those
+resource "local_file" "platform_readmes" {
+  depends_on = [null_resource.copy_template]
+  for_each   = local.platform_readmes
+
+  filename = "${var.output_dir}/docs/platforms/${each.key}"
+  content  = file("${var.foundation_dir}/platforms/${each.key}")
+}
+
+# locals {
+#   guides = try(data.terraform_remote_state.docs["meshstack"].outputs.documentation_guides_md, {})
+# }
+
+# resource "local_file" "meshstack_guides" {
+#   depends_on = [null_resource.copy_template]
+#   for_each   = local.guides
+
+#   filename = "${var.output_dir}/docs/meshstack/guides/${each.key}.md"
+#   content  = each.value
+# }
```

Comment:
do we need it when it is greyed out?

Reply:
will come soon :-) sorry for comitting commented code, that's an artifact of porting over the current code from our internal staging environment that's already running on the docs v2 setup